chapters/VK_KHR_swapchain/wsi.txt

```diff
@@ -93,6 +93,95 @@ include::{generated}/api/protos/vkCreateSwapchainKHR.txt[]
   * pname:pSwapchain is a pointer to a slink:VkSwapchainKHR handle in which
     the created swapchain object will be returned.
 
+As mentioned above, if fname:vkCreateSwapchainKHR succeeds, it will return a
+handle to a swapchain containing an array of at least pname:minImageCount
+presentable images.
+
+While acquired by the application, presentable images can: be used in any
+way that equivalent non-presentable images can: be used.
+A presentable image is equivalent to a non-presentable image created with
+the following slink:VkImageCreateInfo parameters:
+
+[[swapchain-wsi-image-create-info]]
+[options="header"]
+|====
+| sname:VkImageCreateInfo Field | Value
+ifndef::VK_VERSION_1_1,VK_KHR_device_group,VK_KHR_swapchain_mutable_format[]
+| pname:flags                   | 0
+endif::VK_VERSION_1_1,VK_KHR_device_group,VK_KHR_swapchain_mutable_format[]
+ifdef::VK_VERSION_1_1,VK_KHR_device_group,VK_KHR_swapchain_mutable_format[]
+| pname:flags                   |
+ifdef::VK_VERSION_1_1,VK_KHR_device_group[]
+ename:VK_IMAGE_CREATE_SPLIT_INSTANCE_BIND_REGIONS_BIT is set if
+ename:VK_SWAPCHAIN_CREATE_SPLIT_INSTANCE_BIND_REGIONS_BIT_KHR is set
+
+endif::VK_VERSION_1_1,VK_KHR_device_group[]
+ifdef::VK_VERSION_1_1[]
+ename:VK_IMAGE_CREATE_PROTECTED_BIT is set if
+ename:VK_SWAPCHAIN_CREATE_PROTECTED_BIT_KHR is set
+
+endif::VK_VERSION_1_1[]
+ifdef::VK_KHR_swapchain_mutable_format[]
+ename:VK_IMAGE_CREATE_MUTABLE_FORMAT_BIT and
+ename:VK_IMAGE_CREATE_EXTENDED_USAGE_BIT_KHR are both set if
+ename:VK_SWAPCHAIN_CREATE_MUTABLE_FORMAT_BIT_KHR is set
+
+endif::VK_KHR_swapchain_mutable_format[]
+all other bits are unset
+endif::VK_VERSION_1_1,VK_KHR_device_group,VK_KHR_swapchain_mutable_format[]
+| pname:imageType               | ename:VK_IMAGE_TYPE_2D
+| pname:format                  | pname:pCreateInfo->imageFormat
+| pname:extent                  | {pname:pCreateInfo->imageExtent.width, pname:pCreateInfo->imageExtent.height, `1`}
+| pname:mipLevels               | 1
+| pname:arrayLayers             | pname:pCreateInfo->imageArrayLayers
+| pname:samples                 | ename:VK_SAMPLE_COUNT_1_BIT
+| pname:tiling                  | ename:VK_IMAGE_TILING_OPTIMAL
+| pname:usage                   | pname:pCreateInfo->imageUsage
+| pname:sharingMode             | pname:pCreateInfo->imageSharingMode
+| pname:queueFamilyIndexCount   | pname:pCreateInfo->queueFamilyIndexCount
+| pname:pQueueFamilyIndices     | pname:pCreateInfo->pQueueFamilyIndices
+| pname:initialLayout           | ename:VK_IMAGE_LAYOUT_UNDEFINED
+|====
+
+The pname:surface must: not be destroyed until after the swapchain is
```

Comment:
Probably should be `pname:pCreateInfo->surface` at this point in spec. Same with couple others below.

Reply:
Thanks. Still a WIP. I was going to see if I have time this evening to read through the chapter and the change.

Reply:
@stonesthrow There are still some parameters that should have full path, such as `oldSwapchain`.